app/Model/Parcel.php

```diff
@@ -37,6 +37,7 @@
 class Parcel extends Model
 {
     protected $table = 'parcel';
```

Comment:
Parcel is not used for image upload. Numbering should be the object for the photo uploading here.